spec/javascript/utils/compiler/variables.test.js

```diff
@@ -47,6 +79,49 @@ describe("variables.js", () => {
       expect(getVariables(input)).toEqual(expectedOutput)
     })
 
+    describe("Player variable edge cases", () => {
```

Comment:
We can do without this `describe` in my opinion, currently they are used to denote files and function names

Reply:
Would adding a prefix like "Edge-case: " be fine?

Reply:
I don't think it's needed, tests are supposed to catch edge cases by definition. I like the idea, but I'd rather keep the definitions more consistent.